src/Cache/RedisDistributedCache.cs

```diff
@@ -230,28 +203,7 @@ public Task SetAsync<T>(string key, T value, DistributedCacheEntryOptions option
         /// <param name="key">Contains a cache storage key</param>
         public void Refresh(string key)
         {
-            if (string.IsNullOrWhiteSpace(key))
-            {
-                throw new ArgumentNullException(nameof(key));
-            }
-
-            using (var client = this.redisManager.GetClient() as IRedisNativeClient)
-            {
-                if (client.Exists(key) == 1)
-                {
-                    byte[] value = client.Get(key);
-
-                    if (value != null)
-                    {
-                        var expirationValue = client.Get(this.GetExpirationKey(key));
-
-                        if (expirationValue != null)
-                        {
-                            client.Expire(key, int.Parse(Encoding.UTF8.GetString(expirationValue)));
-                        }
-                    }
-                }
-            }
+            // TODO: Implement Sliding expiration
```

Comment:
Is this necessary or do we just want to throw a not implemented exception?

Reply:
I think we can't throw the exception, because Redis makes refresh for us and we aren't call this method to catch exception

Reply:
Sorry, in github view it looks like all code was removed and replaced with a // TODO: